sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -252,6 +248,7 @@ private TracerSdkProvider buildTracerProvider() {
       if (traceConfig != null) {
         tracerProviderBuilder.setTraceConfig(traceConfig);
       }
+      spanProcessors.forEach(tracerProviderBuilder::addSpanProcessor);
```

Comment:
should we also do this up on line 236 to preserve backward compatibility? I was very surprised that there wasn't a unit test that verified this.

Reply:
Thanks, didn't notice / intend the change in behavior.